WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js

```diff
@@ -674,6 +674,9 @@ function(parentIdOrElem, text, width, listener, id) {
 	button.setText(text);
 	button.addSelectionListener(listener);
 	this._replaceControlElement(parentIdOrElem, button);
+	if(className) {
```

Comment:
@tarangkhandelwal, thanks for your suggestion regarding refactoring here. I did same initially. However looking at line: https://github.com/Zimbra/zm-web-client/pull/73/files/57d3b3c1b6908012902f03528496cf577ac5cc5a#diff-89dcba3676fdb8b4880ed597ed2c870dR669 , I changed it to like this to be consistent here. 

Thanks